src/renderer/vanilla/index.js

```diff
@@ -0,0 +1,8 @@
+import { BaseElement } from '../../BaseElement';
+import { StyledElement } from '../../StyledElement';
+import { TemplateElement, html, unsafeHTML, attr } from './TemplateElement';
+import { toString } from '../../util/toString';
+import { defineElement } from '../../util/defineElement';
+import { i18n } from '../../util/i18n';
+
+export { BaseElement, StyledElement, TemplateElement, html, unsafeHTML, attr, toString, defineElement, i18n };
```

Comment:
@lukas-schardt I think this is fine! Although maybe not necessary? Might this be troublesome for bundling and code splitting if someone might use imports from two different places?

I think it would be fine if we only exported the TemplateElement, html and render function from here.

Reply:
It's probably fine! Don't bother with this!

Reply:
But you are right that it may be pretty unnecessary. I put this there so that, if we are switching from lit to vanilla in a project, and we have used the toString or defineElement helper, we can just replace a single import